source/NVDAObjects/UIA/__init__.py

```diff
@@ -1465,6 +1467,17 @@ def event_UIA_controllerFor(self):
 		else:
 			self.event_suggestionsClosed()
 
+	def event_suggestionsOpened(self):
+		super(SearchField, self).event_suggestionsOpened()
+		# #7330: Announce number of items found (except in Start search box where the suggestions are selected as user types).
+		# Oddly, Edge's address omnibar returns 0 for suggestion count when there are clearly suggestions (implementation differences).
+		# Because inaccurate count could be announced (when users type, suggestion count changes), thus announce this if position info reporting is enabled.
+		if config.conf["presentation"]["reportObjectPositionInformation"]:
+			if self.UIAElement.cachedAutomationID == "TextBox" or self.UIAElement.cachedAutomationID == "SearchTextBox" and self.appModule.appName != "searchui":
+				# Item count must be the last one announced.
+				# Translators: presented when there are suggestions as theu ser types (example output: 3 suggestions).
+				queueHandler.queueFunction(queueHandler.eventQueue, ui.message, _("{suggestionsCount} suggestions").format(suggestionsCount = self.controllerFor[0].childCount))

```

Comment:
I'm not sure if this is something we want to braille. The actual suggestion is also displayed with a message, so the suggestion count could suppress that or vice versa. Keep in mind the default message timeout is 4 seconds, so a collision seems likely.
I think showing the index, e.g. 1 of 9, is enough. We already do that. In other words, I would use this message for speech only.